_contentTemplates/common/grid-treelist-autogenerated-columns.md

```diff
@@ -1,4 +1,4 @@
 #grid-treelist-autogenerated-columns-limitation
 When the component initializes, it expects to know what columns to render before the data comes.
-If the component is bound to a collection of dynamic objects (such as `Dictionary` or `ExpandoObject`) due to the nature of such objects and the fact that they can be changed at run time, the component cannot know what columns to render. Therefore, the `AutoGenerateColumns` feature is not applicable in such scenario and should be implemented manually (an example of such setup is available in the [AutoGeneratedColumns page of the Binding a grid to ExpandoObject sample project](https://github.com/telerik/blazor-ui/blob/master/grid/binding-to-expando-object/BindingToExpandoObject/Pages/AutoGeneratedColumns.razor)).
+If the component is bound to a collection of dynamic objects (such as `Dictionary` or `ExpandoObject`) due to the nature of such objects and the fact that they can be changed at run time, the component cannot know what columns to render. Therefore, the `AutoGenerateColumns` feature is not applicable in such scenario and should be implemented manually (an example of such a setup is available in our [knowledge base article on Binding a Grid to ExpandoObject]({%slug grid-kb-binding-to-expando-object%})).
```

Comment:
Simplify the sentence structure. See https://github.com/telerik/blazor-docs/pull/1795